src/app/features/time-tracking/take-a-break/take-a-break.service.ts

```diff
@@ -211,21 +211,24 @@ export class TakeABreakService {
         ico: 'free_breakfast',
         msg,
         translateParams: {
-          time: '15m',
+          time: msToString(cfg.takeABreak.takeABreakSnoozeTime),
         },
         action: {
           label: T.F.TIME_TRACKING.B.ALREADY_DID,
           fn: () => this.resetTimerAndCountAsBreak(),
         },
         action2: {
           label: T.F.TIME_TRACKING.B.SNOOZE,
-          fn: () => this.snooze(),
+          fn: () => this.snooze(cfg.takeABreak.takeABreakSnoozeTime * 60 * 1000),
```

Comment:
The `* 60 * 1000` won't work as intended. The duration field should be already in milliseconds.